lib/config/presets/internal/replacements.ts

```diff
@@ -87,8 +97,20 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
-  'material-ui-to-mui': {
```

Comment:
It depends. If people are referring to `replacements:all`, then no replacement is needed. If people are explicitly referring to `material-ui-to-mui` then yes, we would need a replacement.

I think the likelihood of someone explicitly using this would be low, given replacements aren't even enabled by default.

Reply:
It won't be changed with the recent changes.